llama-index-integrations/embeddings/llama-index-embeddings-huggingface-openvino/llama_index/embeddings/huggingface_openvino/base.py

```diff
@@ -46,10 +47,63 @@ def __init__(
         device: Optional[str] = "auto",
     ):
         self._device = device
-        self._model = model or OVModelForFeatureExtraction.from_pretrained(
-            folder_name, device=self._device, **model_kwargs
-        )
-        self._tokenizer = tokenizer or AutoTokenizer.from_pretrained(folder_name)
+
+        try:
+            from huggingface_hub import HfApi
+        except ImportError as e:
+            raise ValueError(
+                "Could not import huggingface_hub python package. "
+                "Please install it with: "
+                "`pip install -U huggingface_hub`."
+            ) from e
+
+        def require_model_export(
```

Comment:
Too bad having to duplicate this functio, but I agree it's the best option in the context of this PR